recipes/antismash/meta.yaml

```diff
@@ -16,18 +16,18 @@ source:
 
 requirements:
   host:
-    - python >=3
+    - python >=3.6
     - pip
   run:
-    - python >=3
+    - python >=3.6
     - numpy
     - biopython >=1.71
-    - helperlibs
+    - helperlibs >=0.2.1
```

Comment:
Looks like `helperlibs` should be `>=0.2.0`, https://github.com/antismash/antismash/issues/267#issuecomment-636046858. I know there is no version `0.2.0`, but I would still set the version to `>=0.2.0` for consistency with the provided solution.

Reply:
Yes, conda can find `helperlibs` that satisfy the version `>= 0.2.0`.